dpdk_bindings/inlined.c

```diff
@@ -106,9 +106,9 @@ char *rte_pktmbuf_prepend_(struct rte_mbuf *m, uint16_t len)
 struct rte_mbuf *rte_mbuf_from_indirect_(struct rte_mbuf *mi)
 {
     return rte_mbuf_from_indirect(mi);
-} 	
+}
 
 void rte_pktmbuf_detach_(struct rte_mbuf *m)
 {
     rte_pktmbuf_detach(m);
-}
\ No newline at end of file
+}
```

Comment:
Probably best to remove this new line.

Reply:
That's strange! I'm not seeing that new line on my local codebase. I will merge this now, and will recheck if it shows up in the dev branch after merge, will cover it in the next PR if needed.